docs/docs/introduction.md

```diff
@@ -4,10 +4,18 @@ slug: /
 
 # W&B Weave
 
-:::tip
-For a limited time, the new W&B Inference service is included in your free tier. W&B Inference provides access to leading open-source foundation models via API and the Weave [Playground](./guides/tools/playground.md). 
-- [Developer documentation](./guides/integrations/inference.md)
-- [Product page](https://wandb.ai/site/inference) 
+:::tip[🚀 New to Weave? Start with W&B Inference!]
```

Comment:
Shall we get rid of all the emojis and emdashes?

Reply:
Removed this bit